backend/.env.example

```
@@ -1,14 +1,28 @@
 ENVIRONMENT=development
-OPENAI_API_KEY=your-openai-api-key-here
-OPENAI_ORGANIZATION=your-openai-organization-here
+BACKEND_CORS_ORIGINS=[]
+
+API_V1_STR=api/v1
 
+# DATABASE
+DB_USERNAME=genai
+DB_PASSWORD=genai123
+DB_HOST=postgres
+DB_NAME=genai_db 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)